package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "test": "mocha",
-    "build": "rollup -c -f cjs -o dist/pluginutils.cjs.js && rollup -c -f es6 -o dist/pluginutils.es6.js",
+    "build": "rollup -c -f cjs -o dist/pluginutils.cjs.js && rollup -c -f es -o dist/pluginutils.es.js",
```

Comment:
What about `jsnext:main` field? Also there's now also `module` field which is the sam.e